src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Venues/Delete.cshtml

```diff
@@ -0,0 +1,67 @@
+﻿@model Dfc.CourseDirectory.WebV2.Features.DataManagement.Venues.Delete.Response
```

Comment:
Could we call this view `DeleteRow.cshtml` so it's clearly differentiated between `DeleteUpload.cshtml`?

Reply:
done